test/performance/benchmarks/channel-pubsub/continuous/200-channel-perf.yaml

```diff
@@ -0,0 +1,135 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: channel-perf-send-receive
+  namespace: default
+  labels:
+    role: channel-perf-consumer
+spec:
+  schedule: "0/15 * * * *"
+  # History must be zero to ensure no failed pods stick around and block the next job
+  successfulJobsHistoryLimit: 0
+  failedJobsHistoryLimit: 0
+  jobTemplate:
+    spec:
+      completions: 1
+      parallelism: 1
+      backoffLimit: 0
+      # Allow up to 14 minutes, then clean up to make room for the next attempt
+      activeDeadlineSeconds: 840
+      template:
+        metadata:
+          labels:
+            role: channel-perf-consumer
+        spec:
+          serviceAccountName: perf-pubsub
+          restartPolicy: Never
+          containers:
+            - name: sender-receiver
+              image: knative.dev/eventing/test/test_images/performance
+              args:
+                - "--roles=sender,receiver"
+                # FIXME: not sure if this address is correct
```

Comment:
I think this is correct, remove the FIXME

Reply:
Removed.